R/data-converters.R

```diff
@@ -370,9 +378,14 @@ enw_aggregate_cumulative <- function(
   init_ref_date <- min_reference_date + internal_timestep - 1
   agg_obs <- obs[report_date >= init_ref_date]
 
-  stopifnot(
-    "There are no complete report dates (i.e. report_date >= min_date + timestep - 1)" = nrow(agg_obs) > 0 # nolint
-  )
+  if (nrow(agg_obs) == 0) {
+    cli::cli_abort(
+      c(
+        "There are no complete report dates",
```

Comment:
can this really not just be one line?

Reply:
counter: the next line should probably give parsed values. So maybe more like
There are no rows with report_date >= min_date + timestep - 1
i.e. report_date >= {min_date + timestep - 1}

However: this is drifting into write-the-style-guidance-then-fix new PR territory.